### DISCUSSION TITLE
Edge Cookie Editor: please add "Cookie Write Protection" (as per Cookie Quick Manager)

### DISCUSSION THREAD

**Discussion by Klipperken** [community/community#38398] [Other Feature Feedback, Questions, & Ideas]
Sir, could you, as per the title ... :
Edge Cookie Editor: please add "Cookie Write Protection" (as per Cookie Quick Manager)

Thank you very much.

**Comment by jfullstackdev** [comment]
you mean you are using Microsoft Edge ? and that prompted you ?

**Comment by Klipperken** [comment]
No, sir, as in: Edge Cookie Editor does not have a Cookie Write Protection?
Often certain cookies are altered, though this dares screwing me use of the
browser up.
